helm/sematic-grafana-dashboards/templates/sematic_runs_dashboard_configmap.yaml

```
@@ -0,0 +1,637 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    grafana_dashboard: "1"
+  name: grafana-dashboard-sematic-runs
+data:
+  sematic_runs_dashboard_configmap.json: |-
+    {
+      "annotations": {
+        "list": [
+          {
+            "builtIn": 1,
+            "datasource": {
+              "type": "{{ .Values.datasources.grafana.type }}",
+              "uid": "{{ .Values.datasources.grafana.uid }}"
+            },
+            "enable": true,
+            "hide": true,
+            "iconColor": "rgba(0, 211, 255, 1)",
+            "name": "Annotations & Alerts",
+            "target": {
+              "limit": 100,
+              "matchAny": false,
+              "tags": [],
+              "type": "dashboard"
+            },
+            "type": "dashboard"
+          }
+        ]
+      },
+      "editable": true,
+      "fiscalYearStartMonth": 0,
+      "graphTooltip": 0,
+      "id": null,
+      "links": [],
+      "liveNow": false,
+      "panels": [
+        {
+          "datasource": {
+            "type": "{{ .Values.datasources.prometheus.type }}",
+            "uid": "{{ .Values.datasources.prometheus.uid }}"
+          },
+          "gridPos": {
+            "h": 3,
+            "w": 24,
+            "x": 0,
+            "y": 0
+          },
+          "id": 3,
+          "options": {
+            "content": "# Run ID\n\n# [$run_id]({{ .Values.sematic_server_dashboard_url }}/runs/$run_id)",
+            "mode": "markdown"
+          },
+          "pluginVersion": "9.1.7",
+          "type": "text"
+        },
+        {
+          "datasource": {
+            "type": "{{ .Values.datasources.loki.type }}",
+            "uid": "{{ .Values.datasources.loki.uid }}"
+          },
+          "gridPos": {
+            "h": 10,
+            "w": 24,
+            "x": 0,
+            "y": 3
+          },
+          "id": 1,
+          "options": {
+            "dedupStrategy": "none",
+            "enableLogDetails": true,
+            "prettifyLogMessage": false,
+            "showCommonLabels": false,
+            "showLabels": false,
+            "showTime": false,
+            "sortOrder": "Descending",
+            "wrapLogMessage": false
+          },
+          "targets": [
+            {
+              "datasource": {
+                "type": "{{ .Values.datasources.loki.type }}",
+                "uid": "{{ .Values.datasources.loki.uid }}"
+              },
+              "editorMode": "code",
+              "expr": "{job=~\".*$run_id.*\", app != \"sematic-server\", namespace=\"$namespace\"} |= `` |= `$log_filter`",
```

```suggestion
              "expr": "{job=~\".*worker-$run_id.*\", app != \"sematic-server\", namespace=\"$namespace\"} |= `` |= `$log_filter`",
```

Probably want to do this, otherwise when the run id is empty we will also get *server* logs in the logs viewer.

---

fixed